src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -959,6 +960,57 @@ def _configure_security_policy_storage_params(arg_ctx):
         c.argument('allow_data_loss',
                    arg_type=allow_data_loss_param_type)
 
+    ###############################################
+    #             sql instance pool               #
+    ###############################################
+
+    with self.argument_context('sql instance-pool') as c:
+        c.argument('instance_pool_name', options_list=['--name', '-n'],
+                   help="Instance Pool Name")
+        c.argument(
+            'tier',
+            arg_type=tier_param_type,
+            required=True,
+            help='The edition component of the sku. Allowed values: GeneralPurpose, BusinessCritical.')
+
+        c.argument('family',
+                   arg_type=family_param_type,
+                   required=True,
+                   help='The compute generation component of the sku. '
+                   'Allowed values include: Gen4, Gen5.')
+
+    with self.argument_context('sql instance-pool create') as c:
+        # Create args that will be used to build up the InstancePool object
+        create_args_for_complex_type(
+            c, 'parameters', InstancePool, [
+                'location',
+                'license_type',
+                'subnet_id',
+                'vcores',
+                'tags'
+            ])
+
+        c.argument('vcores',
+                   required=True,
+                   arg_type=capacity_param_type,
+                   help='the capacity of the instance pool in vcores.')
+
+        c.argument(
+            'subnet_id',
+            options_list=['--subnet'],
+            required=True,
+            help='ID of the subnet that allows access to an Azure Sql Instance Pool.')
```

Comment:
I understand you based this on the existing help text, but it sounds weird to me. Can it be rephrased? How would you explain this to somebody new to instance pool?